src/core/credrank/markovProcessGraph.test.js

```diff
@@ -29,18 +29,30 @@ describe("core/credrank/markovProcessGraph", () => {
   const na = (name) => NA.fromParts([name]);
   const ea = (name) => EA.fromParts([name]);
 
-  const participantNode = {
-    description: "participant",
-    address: na("participant"),
+  const participantNode1 = {
+    description: "participant1",
+    address: na("participant1"),
     timestampMs: null,
   };
   const id1 = uuid.fromString("YVZhbGlkVXVpZEF0TGFzdA");
-  const participant = {
-    description: participantNode.description,
-    address: participantNode.address,
+  const participant1 = {
+    description: participantNode1.description,
+    address: participantNode1.address,
     id: id1,
   };
 
+  const participantNode2 = {
+    description: "participant2",
+    address: na("participant2"),
+    timestampMs: null,
+  };
+  const id2 = uuid.fromString("YVZhbGlkVXVpZE20TGFzdA");
+  const participant2 = {
+    description: participantNode2.description,
+    address: participantNode2.address,
+    id: id2,
+  };
+
```

Comment:
Added a second participant because during implementation I found a bug that only repro'd with multiple participants (had to do with the whether an assignment after all participants or in between participants--see above). I believe adding this makes the test suite generally more effective, it also means we have a participant with no contributions, which is realistic.